src/sasquatchbackpack/cli.py

```diff
@@ -0,0 +1,176 @@
+        issues.append(
+            f"Your provided duration ({str(total_duration)}) is too "
+            + "small. The minimum is 1 hour."
+        )
+
+    if radius > 5000:
+        issues.append(
+            f"Your provided radius ({radius}) is too large. The maximum"
+            + " is 5000."
+        )
+    elif radius <= 0:
+        issues.append(
+            f"Your provided radius ({radius}) is too small. The minimum"
+            + " is 0.1."
+        )
+
+    if coords[0] < -90.0:
+        issues.append(
+            f"Your provided latitude ({coords[0]}) is too low. The"
+            + " minimum is -90."
+        )
+    elif coords[0] > 90.0:
+        issues.append(
+            f"Your provided latitude ({coords[0]}) is too high. The"
+            + " maximum is 90."
+        )
+
+    if coords[1] < -180:
+        issues.append(
+            f"Your provided longitude ({coords[1]}) is too low. The"
+            + " minimum is -180."
+        )
+    elif coords[1] > 180:
+        issues.append(
+            f"Your provided longitude ({coords[1]}) is too high. The"
+            + " maximum is 180."
+        )
+
+    if min_magnitude < 0:
+        issues.append(
+            f"Your provided minimum magnitude ({min_magnitude}) is too"
+            + " small. The minimum is 0."
+        )
+    elif min_magnitude > 10:
+        issues.append(
+            f"Your provided minimum magnitude ({min_magnitude}) is too"
+            + " large. The maximum is 10."
+        )
+
+    if max_magnitude > 10:
+        issues.append(
+            f"Your provided maximum magnitude ({max_magnitude}) is too"
+            + " large. The maximum is 10."
+        )
+    elif max_magnitude < 0:
+        issues.append(
+            f"Your provided maximum magnitude ({max_magnitude}) is too"
+            + " small. The minimum is 0."
+        )
+
+    if min_magnitude > max_magnitude:
+        issues.append(
+            f"Your provided minimum magnitude ({min_magnitude}) cannot"
+            + f" excede your provided maximum magnitude ({max_magnitude})."
+        )
+
+    if len(issues) > 0:
+        click.secho(
+            "Command Failed! One or more provided parameters is"
+            + "incorrect.\n",
+            fg="red",
+        )
+        click.echo("FAILURE\n------")
+        for issue in issues:
+            click.echo(issue)
+        click.echo("------")
+        return
+
+    results = usgs.search_api(
+        total_duration,
+        radius,
+        coords,
+        min_magnitude,
+        max_magnitude,
+    )
+
+    if len(results) > 0:
+        click.secho("SUCCESS!", fg="green")
+        click.echo("------")
+        for result in results:
+            click.echo(result)
+        click.echo("------")
+    else:
+        click.secho("SUCCESS", fg="orange")
+        click.echo("------")
+        click.echo("No results found for the provided criteria :(")
+        click.echo("------")
+
+
+if __name__ == "__main__":
```

Comment:
You can remove this since you have  `main()` function in `[project.scripts]` in your `pyproject.toml` to make this an executable script.